low-level/framework/base/sspl_constants.py

```diff
@@ -79,6 +79,10 @@
 SSPL_CONFIG_INDEX = "SSPL"
 SSPL_TEST_CONFIG_INDEX = "SSPL_TEST"
 CONFIG_SPEC_TYPE = "yaml"
+DEFAULT_DC = "DC01"
+DEFAULT_RACK = "RC01"
+DEFAULT_SN = "SN01"
+DEFAULT_CLUSTER = "CC01"
```

Comment:
These are required keys. No need to have default values.

Reply:
> self._site_id = Conf.get(GLOBAL_CONF, SITE_ID_KEY,'DC01')
All the sensor files keeps the default values with Conf api as of today.
similar mapping is done in conf_dict.
any change required?

Reply:
These default values comes into picture only when config read failed, using conf store apis.  see @mariyappanp's point that if these values are used (in case of erro), alerts would be generated using wrong values. On other hand, we can error out saying config read failed, we cant generate alert.

What is more imp? Generating alerts for sure, so SSPL should, but with caveats these values seen are not actual ones. So feed the values accordingly, like DEFAULT_DC01, & so on. So if any alerts show these values, we know what would have happened and check in sspl logs.

Reply:
@cdeshmukh  @mariyappanp please confirm

Reply:
Ack.
Please check it is in the correct format or not?

Reply:
Am fine